shibboleth/app_settings.py

```diff
@@ -30,3 +30,8 @@
 #LOGOUT_REDIRECT_URL specifies a default logout page that will always be used when
 #users logout from Shibboleth.
 LOGOUT_REDIRECT_URL = getattr(settings, 'SHIBBOLETH_LOGOUT_REDIRECT_URL', None)
+#This is the parameter used to redirect after login. Old versions was used 'next' parameter.
+LOGIN_URL_PARAMETER = getattr(settings, 'SHIBBOLETH_LOGIN_URL_PARAMETER', 'target')
+#This should look like: /login
+#This is the LOGIN_URL_PARAMETER value. The endpoint after login
+LOGIN_REDIRECT_URL = getattr(settings, 'SHIBBOLETH_LOGIN_REDIRECT_URL', None)
```

Comment:
Tests for these LOGIN settings would be nice.

Reply:
Ok, I will do.